es-curator_v1beta1.yaml

```diff
@@ -10,8 +10,11 @@ spec:
         spec:
           containers:
           - name: curator
-            image: bobrik/curator
-            args: ["--config", "/etc/config/config.yml", "/etc/config/action_file.yml"]
+            image: praseodym/elasticsearch-curator:5.4.1
```

Comment:
Why change the image? Maybe I should provide my own?

Reply:
It is an automated, up to date build of curator. The previous image was not up to date and could not really pinned.
Not sure it's worth maintaining an image.

Reply:
Please, try `quay.io/pires/docker-elasticsearch-curator:5.4.1`.

Reply:
Have you tried it @mindw ?

Reply:
yes